bin/stencil-download.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+require('colors');
+const apiHost = 'https://api.bigcommerce.com';
+const dotStencilFilePath = './.stencil';
+const options = { dotStencilFilePath };
+const pkg = require('../package.json');
+const Program = require('commander');
+const stencilDownload = require('../lib/stencil-download');
+const versionCheck = require('../lib/version-check');
+const themeApiClient = require('../lib/theme-api-client');
+const inquirer = require('inquirer');
+
+Program
+    .version(pkg.version)
+    .option('--host [hostname]', 'specify the api host', apiHost)
+    .option('--file [filename]', 'specify the filename to download only')
+    .option('--type [type]', 'specify the type of content to download available are web-pages, theme, web-dav')
```

Comment:
I don't see the type being used

Reply:
@junedkazi - This shouldn't be part of the option here and it is removed!